pyriemann/utils/distance.py

```diff
@@ -133,6 +133,23 @@ def distance(A, B, metric='riemann'):
 
     return d
 
+def pairwise_distance(X, Y=None, metric='riemann'):
+    """Pairwise distance matrix"""
```

Comment:
While you are at it, can you add docstring about the arguments.
I probably forgot to do it myself.